doc/examples/filters/plot_inpaint.py

```diff
@@ -43,12 +43,12 @@
 
 fig, axes = plt.subplots(ncols=3, nrows=1)
 
-axes[0].set_title('Defected image')
+axes[0].set_title('Original image')
 axes[0].imshow(image_orig)
 axes[0].set_xticks([]), axes[0].set_yticks([])
 
-axes[1].set_title('Defect mask')
-axes[1].imshow(mask, cmap=plt.cm.gray)
```

Comment:
I found it more representative to show defect mask, not defected image.
We might consider adding `defected image` between `defected mask` and `inpainted image`, so creating 2x2 matrix of images.